SwiftCompilerSources/Sources/SIL/Function.swift

```diff
@@ -59,6 +63,10 @@ final public class Function : CustomStringConvertible, HasShortDescription, Hash
     entryBlock.arguments.lazy.map { $0 as! FunctionArgument }
   }
 
+  public func argument(at: Int) -> FunctionArgument {
```

Comment:
nitpick: `at` is not a good name for the parameter value -> `public func argument(at index: Int) -> FunctionArgument`